pipeline/vars/lib.groovy

```diff
@@ -215,4 +217,73 @@ def SendUMBMessage(def msgMap, def overrideTopic, def msgType){
 
 }
 
+def sendEmail(def testResults, def artifactDetails, def tierLevel){
+    /*
+        Send an Email
+        Arguments:
+            testResults: map of the test suites and its status
+                Example: testResults = [ "01_deploy": "PASS", "02_object": "PASS"]
+            artifactDetails: Map of artifact details
+                Example: artifactDetails = ["composes": ["rhe-7": "composeurl1",
+                                                         "rhel-8": "composeurl2"],
+                                            "product": "Redhat",
+                                            "version": "RHCEPH-5.0",
+                                            "ceph_version": "16.2.0-117",
+                                            "repository": "reponame"]
+            tierLevel:
+                Example: Tier0, Tier1, CVP..
+    */
+    def status = "STABLE"
+    def toList = "ceph-qe-list@redhat.com"
+    def body = readFile(file: "pipeline/vars/emailable-report.html")
+
```

Comment:
IMO, we should be adding `<body>` to body before creating the table. That is
```
body += "<body>"
body += "<u><h3>Artifacts</h3></u>
body += "<table>"
if (artifactDetails.composes) {
  body += "<tr><td>Composes</td><td>${artifactDetails.composes}</td></tr>"
}
...
```

Reply:
Please change the order to
- Product
- Version
- Ceph version
- Composes
- Image

Reply:
done